kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -321,14 +328,28 @@ public void stop(boolean clean) {
             if (clean) {
                 repository.clean();
             }
-            log.infof("KieServerRouter stopped on %s:%s at %s",
-                      System.getProperty(ROUTER_HOST),
-                      System.getProperty(ROUTER_PORT),
-                      new Date());
+            if (log.isInfoEnabled()) {
+                logServerInfo("KieServerRouter stopped on ", System.getProperty(ROUTER_HOST),
+                        Integer.getInteger(ROUTER_PORT), Integer.getInteger(ROUTER_PORT_TLS));
+            }
+            
+            
         } else {
             log.error("KieServerRouter was not started");
         }
     }
+    
+    private void logServerInfo(String prefix, String host, int port, int portTls) {
+        StringBuilder sb = new StringBuilder(prefix);
+        if (isHttpEnabled) {
+            sb.append(host).append(':').append(port);
+        }
+        if (TLS_ENABLED) {
+            sb.append(host).append(':').append(portTls).append(" (TLS)");
+        }
+        log.info(sb);
```

Comment:
if both `http` and `https/tls` are enabled, the output shown by this log might be a little difficult to read - am also missing the `date` as part of the output in the original code though it might be redundant as the log may display the date.

Reply:
I slightly change the log to avoid that issue. Date was clearly redundant with log date and I intentionally remove it